sdk/test/management-sdk.ts

```diff
@@ -11,7 +11,7 @@ var manager: AccountManager;
 describe("Management SDK", () => {
 
     beforeEach(() => {
-        manager = new AccountManager(/*serverUrl=*/ "http://localhost", /*userAgent=*/ "unit-test/1.0.0");
+        manager = new AccountManager(/*accessKey=*/ "stub", /*userAgent=*/ "unit-test/1.0.0", /*serverUrl=*/ "http://localhost");
```

Comment:
Perhaps some form of "access key" instead of "stub."  A stub returns a canned response.  The response itself is not a stub, so having AccountManager.accessKey return "stub" is odd.